index.d.ts

```diff
@@ -12,6 +12,7 @@ interface FontAwesomeIconProps {
   pull?: 'right' | 'left'
   pulse?: boolean
   rotation?: 90 | 180 | 270 | '90' | '180' | '270'
+  rotateBy?: boolean
```

Comment:
As reference, our docs show using `rotateBy` like the following:
```
 <i class="fa-solid fa-snowboarding fa-rotate-by" style="--fa-rotate-angle: 80deg;" ></i>
```

Reply:
Hmm, we might want to chat about this one. Because this is a _little_ awkward.

For folks to use this do they have to first set `rotate-by` on their component and then use `:style="{ '--fa-rotate-by': '80deg' }` as well? I wonder if instead of this we can detect when they are trying to use `--fa-rotate-by` and automatically apply the `fa-rotate-by` class when we see it?

Or...do you feel like this explicit nature (even if it is two steps) is worth it for clarity?

Reply:
we discussed internally and the decision was made to leave as is... the more explicit way, which matches our docs.